cmd/metacache-set.go

```diff
@@ -941,12 +941,12 @@ func listPathRaw(ctx context.Context, opts listPathRawOptions) (err error) {
 			}
 			w.CloseWithError(werr)
 
-			if werr != io.EOF && werr != nil &&
+			if werr != nil && werr.Error() != io.EOF.Error() &&
 				werr.Error() != errFileNotFound.Error() &&
 				werr.Error() != errVolumeNotFound.Error() &&
 				werr.Error() != errDiskNotFound.Error() &&
 				werr.Error() != errUnformattedDisk.Error() &&
-				!errors.Is(werr, context.Canceled) {
+				werr.Error() != context.Canceled.Error() {
```

Comment:
```suggestion
				werr.Error() != context.Canceled.Error() &&
				!errors.Is(werr, context.Canceled) {
```

Otherwise we start printing wrapped context canceled arrors.

Reply:
It is changed on purpose because errors.Is() didn't ignore it @klauspost - I wouldn't have changed it otherwise

Reply:
PTAL @klauspost it may be that something is changing underneath the context.Canceled error.

Reply:
I actually completely removed this log since it's not useful.